api/mod/tests/conftest.py

```diff
@@ -8,14 +9,37 @@
 import alembic
 import pytest
 from alembic.config import Config
-from sqlalchemy import create_engine
+from fastapi.testclient import TestClient
 from sqlalchemy.orm import scoped_session, sessionmaker
 
 from mod.src.app import APP
-from mod.src.database.database import get_db
+from mod.src.database.database import engine, get_db
 from mod.src.settings import SETTINGS
 
 
+def setup_test_db() -> None:
+    """Apply migrations to test database and load test data"""
```

Comment:
```suggestion
    """Apply migrations to test database and load test data. Is executed exactly once before the tests are run. Needs no cleanup because the database lies in a temporary folder."""
```